readme.md

```diff
@@ -127,10 +127,10 @@
 	- [Observables](https://github.com/sindresorhus/awesome-observables#readme)
 	- [npm scripts](https://github.com/RyanZim/awesome-npm-scripts#readme) - Task runner.
 	- [30 Seconds of Code](https://github.com/30-seconds/30-seconds-of-code#readme) - Code snippets you can understand in 30 seconds.
-- [Swift](https://github.com/matteocrippa/awesome-swift#readme)
+- [Swift](https://github.com/matteocrippa/awesome-swift#readme) -  Compiled programming language for iOS, macOS, watchOS, tvOS, and Linux applications.
```

Comment:
This should rather describe what Swift is than mention where it runs, as it also runs on Windows now and more in the future.